pkgs/desktops/gnome/extensions/system-monitor/default.nix

```diff
@@ -17,6 +17,15 @@ stdenv.mkDerivation rec {
   ];
 
   patches = [
+    # GNOME 43 compatibility
+    (fetchpatch {
+      url = "https://github.com/paradoxxxzero/gnome-shell-system-monitor-applet/pull/754/commits/3b3a617f78c5e9bbce0aa2864b3989335edb37b7.patch";
+      hash = "sha256-BAD0ExQYmTg6i02Gg7jcF0vf1zp232vXRjGCn+rBYXE=";
+    })
+    (fetchpatch {
```

Comment:
Could we use the entire PR as a single patch? I know that this is possible in theory, just not sure about how versioning / pinning works in that case

Reply:
There is https://github.com/paradoxxxzero/gnome-shell-system-monitor-applet/pull/754.patch, but that’s only useful to us for a *merged* PR, as any update would invalidate its hash.